source/Quantities/Creation/Creators.cs

```diff
@@ -14,13 +16,9 @@ namespace Quantities.Creation;
     public Quotient<TUnit, TDenominator> Per<TDenominator>(in Scalar<TDenominator> denominator)
         where TDenominator : IUnit, IDimension => new(denominator.factory.Inject(this.factory.Quotient));
     internal Quantity Create(in Double value) => new(in value, this.factory.Create());
-    internal Quantity Create<TAlias, TLinear>(in Double value)
-        where TAlias : TUnit, IAlias<TLinear>
-        where TLinear : ILinear, IDimension => new(in value, this.factory.AliasTo<TAlias, TLinear>());
+    internal Quantity Create(in Double value, ChooseMeasure choose) => new(in value, choose(this.factory));
```

Comment:
Is this Vorarbeit for normalisation? I don't understand what you gain by letting each measure specify their factory.

Reply:
Sorry, very bad naming on my side. It doesn't choose the factory, it selects how to create the measure instance. I'll rename both the delegate type and parameter name to something more readable.

Reply:
Now I read it that it's injected with a delegate factory, which makes sense now. Thank you 😊